components/bootloader_support/src/bootloader_init.c

```diff
@@ -45,6 +45,13 @@ esp_err_t bootloader_check_bootloader_validity(void)
     unsigned int major = revision / 100;
     unsigned int minor = revision % 100;
     ESP_EARLY_LOGI(TAG, "chip revision: v%d.%d", major, minor);
+
+#if defined(CONFIG_SOC_SERIES_ESP32) && !defined(CONFIG_ESP32_USE_UNSUPPORTED_REVISION)
+    if (major < 3) {
+        ESP_EARLY_LOGE(TAG, "Unsupported chip revision!");
```

Comment:
What about also adding something like:
```"This ESP32 SoC uses an unsupported revision. Please enable CONFIG_ESP32_USE_UNSUPPORTED_REVISION to allow its usage."```

Reply:
Even with the `CONFIG_ESP32_USE_UNSUPPORTED_REVISION=y`, I guess it can also print a warning message.